app/views/posts/index.html.erb

```diff
@@ -12,7 +12,7 @@
         <div class="post-comment-box">
           <card class="post-item">
             <div class="post-data">
-              <h2>Post <%= post.id %></h2>
+              <h2> <%= post.title %></h2>
               <p> <%= post.text %> </p>
               <div class="counter-data">
                 <h5 class="custom-card-p">Comments: <%= post.comments_counter %> </h5>
```

Comment:
- [OPTIONAL] For a great user experience, I would advise you to include a button for creating a new post on the `Post#Index` page. This would make it easy for a user to create a post and they don't have to start navigating back to the `User#Index` page to create a post. Kindly think about it and implement it. 👍 😎